devices/1.json

```diff
@@ -1,6 +1,8 @@
 {"mdl":"Monitor Plug",
     "parameters":[
-        {"id": "REAL_POWER",    "component":"sensor", "stat_t":"~@/state","config": {"dev_cla":"power","unit_of_meas":"W"}},
-        {"id": "REACTIVE_POWER","component":"sensor", "stat_t":"~@/state","config": {"dev_cla":"reactive_power","unit_of_meas":"var"}}
+        {"id": "REAL_POWER",    "component":"sensor", "stat_t":"~@/state","config": {"name":"Power","dev_cla":"power","unit_of_meas":"W","enabled_by_default":"true"}},
+        {"id": "REACTIVE_POWER","component":"sensor", "stat_t":"~@/state","config": {"name":"Reactive Power","dev_cla":"reactive_power","unit_of_meas":"var","enabled_by_default":"true"}},
+        {"id": "FREQUENCY",     "component":"sensor", "stat_t":"~@/state","config": {"name":"Frequency","value_template":"{{ value|round(2) }}","dev_cla":"frequency","unit_of_meas":"Hz","enabled_by_default":"true"}},
+        {"id": "VOLTAGE",     "component":"sensor", "stat_t":"~@/state","config": {"name":"Voltage","dev_cla":"voltage","unit_of_meas":"V","enabled_by_default":"true"}}
```

Comment:
`enabled_by_default` should only be specified when not default (true). It should be specified as `false` for Voltage and Frequency for backwards compatibility.
Also, I'm not sure about `name` being specified here; it ends up being defined twice for the entities in the discovery message. `app.js` already adds it (albeit lowercase at present - but I have a fix for that :) )

Reply:
I've pushed the change to develop for the entity names to be auto-constructed Title Case (from lower case), this should remove your need for the "friendly names" in the .json files.

Reply:
> I've pushed the change to develop for the entity names to be auto-constructed Title Case (from lower case), this should remove your need for the "friendly names" in the .json files.

There was a but with your code, but I have fixed it to also auto-constructed Title Case from the lower case words.